model-optimizer/extensions/ops/BatchNormInference.py

```diff
@@ -18,18 +18,23 @@
 from mo.ops.op import Op
 
 
-class BNOp(Op):
+class BatchNormInference(Op):
     """
     Empty Op for BN layer. It will be replaced by BNToScaleShift FrontReplacer
     """
-    op = 'BN'
-    enabled = True
+    op = 'batchNormInference'
+    enabled = False
 
     def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, {
-            'type': None,
-            'op': __class__.op,
+            'type': self.op,
```

Comment:
Please, leave `type` equal to `None` for now. We don't want to emit IR with this operation yet.

Reply:
fixed